.github/workflows/actions.yml

```diff
@@ -124,8 +129,8 @@ jobs:
         run: make maintainer-clean
 
   trexio_macos:
-    name: x86 MacOS 12
-    runs-on: macos-12
+    name: arm64 MacOS
+    runs-on: macos-14
```

Comment:
Is it possible to keep the x86 also? There are still people with x86 Macs around...

Reply:
Will do

Reply:
Actually for testing it does not matter if we are on x86 or arm64, it is only important for release CI. I prefer to test on arm64 MacOS because most Mac users have already moved to these machines.